docs/install.md

```diff
@@ -20,13 +20,15 @@ yarn add --dev --exact prettier
 
 <!--END_DOCUSAURUS_CODE_TABS-->
 
-Then, create an empty config file to let editors and other tools know you are using Prettier:
+Then, create an empty `prettier.config.js` file to let editors and other tools know you are using Prettier and add the content below:
+```js
+/** @type {import('prettier').Config} */
+module.exports = {
 
-<!-- Note: `echo "{}" > .prettierrc.json` would result in `"{}"<SPACE>` on Windows. The below version works in cmd.exe, bash, zsh, fish. -->
-
-```bash
-echo {}> .prettierrc.json
+}
 ```
+This way, the editor will autocomplete the options you have available according to your installed version of Prettier.
```

Comment:
Shouldn't editor also do autocomplete to json file too? https://github.com/SchemaStore/schemastore/blob/master/src/schemas/json/prettierrc.json

Reply:
JSON files don’t allow `require("some-prettier-plugin")` and not using `require` does not work in Yarn PnP. On the other hand `/** @type {import('prettier').Config} */`, only works if [`@types/prettier`](https://www.npmjs.com/package/@types/prettier) is installed.

Reply:
> On the other hand `/** @type {import('prettier').Config} */`, only works if [`@types/prettier`](https://www.npmjs.com/package/@types/prettier) is installed.

Great observation, maybe this should be added in the docs also?

Reply:
![image](https://user-images.githubusercontent.com/172584/204206097-ddd48433-790e-4b1d-b0d0-ecebc2cff834.png)
![image](https://user-images.githubusercontent.com/172584/204206163-4ba68d07-fc7c-4090-b85d-0960762d0b97.png)

Reply:
This only works with JSON files. With `.prettierrc.js`, `prettier.config.cjs` etc this does not show errors:

<img width="266" alt="Screenshot 2022-12-08 at 13 23 48" src="https://user-images.githubusercontent.com/1935696/206445606-1602f94e-c26b-461b-b89e-b999d2cf018a.png">

Opened a followup PR for this: